packages/ag-charts-community/src/options/chart/zoomOptions.ts

```diff
@@ -1,8 +1,13 @@
+export type AgZoomAnchorPoint = 'pointer' | 'start' | 'end';
 export type AgZoomAxes = 'x' | 'y' | 'xy';
 export type AgZoomPanKey = 'alt' | 'ctrl' | 'meta' | 'shift';
-export type AgZoomScrollingPivot = 'pointer' | 'start' | 'end';
 
 export interface AgZoomOptions {
+    /** The anchor point about which to zoom into when scrolling, defaults to `end`. */
+    anchorPoints?: {
+        x: AgZoomAnchorPoint;
+        y: AgZoomAnchorPoint;
+    };
```

Comment:
Would it be preferable to make this `xAnchorPoint` and `yAnchorPoint` instead? What do we usually do for things like this?

Reply:
It maybe worth inverting the nesting, if there are several properties relating to each axis direction?

e.g.:
```
  xAxis: AgZoomAxisOptions,
  
  ....
  
interface AgZoomAxisOptions {
  enabled?: boolean; // Replaces `axes?: AgZoomAxes`?
  minRatio?: number;
  anchorPoint?: AgZoomAchorPoint;
}
```

Reply:
Maybe something to consider discussing with Rob + David ^